pkg/api/team_members.go

```diff
@@ -50,9 +58,48 @@ func AddTeamMember(c *m.ReqContext, cmd m.AddTeamMemberCommand) Response {
 	})
 }
 
+// PUT /:teamId/members/:userId
+func UpdateTeamMember(c *m.ReqContext, cmd m.UpdateTeamMemberCommand) Response {
```

Comment:
There seems to be a bit of a mix throughout the API, those who need access to something on the HTTPServer are instances and others aren't. This one doesn't need access to anything on the instance, should it still be an instance method?

Reply:
It's just that we have not migrated all to instance methods. Think for consistency they should. This function could then use the bus on the instance instead of the global one.

Reply:
That's a good catch, hadn't even thought of the bus access.